tests/Utility/OrganizationFeaturesTest.php

```diff
@@ -42,6 +44,34 @@ public function testOrganizationFeatures(?string $propertyValue, bool $expectedR
         $this->assertEquals($expectedResult, OrganizationFeatures::isMonetizationEnabled($organization));
     }
 
+    /**
+     * Test if an organization has pagination enabled.
+     */
+    public function testPaginationEnabled(): void
+    {
+        /** @var \Apigee\Edge\Api\Management\Entity\OrganizationInterface $organization */
+        $organization = $this->getMockBuilder(OrganizationInterface::class)->getMock();
+        $organization->method('getPropertyValue')->will($this->returnValueMap([
+            ['features.isCpsEnabled', null],
+            ['features.hybrid.enabled', 'true'],
+        ]));
+        $this->assertTrue(OrganizationFeatures::isPaginationEnabled($organization));
+
+        $organization = $this->getMockBuilder(OrganizationInterface::class)->getMock();
+        $organization->method('getPropertyValue')->will($this->returnValueMap([
+            ['features.isCpsEnabled', 'true'],
+            ['features.hybrid.enabled', null],
```

Comment:
It would be a more realistic test if you remove this line since an org that is not hybrid will not return the `features.hybrid.enabled` at all:

```
GET https://api.enterprise.apigee.com/v1/o/myorg

{
  ...
  "name": "myorg",
  "properties": {
    "property": [
      {
        "name": "features.isSmbOrganization",
        "value": "true"
      },
      {
        "name": "features.isCpsEnabled",
        "value": "true"
      }
    ]
  },
  "type": "paid"
}
```

Reply:
`OrganizationFeatures::isFeatureEnabled()` will read the `features.hybrid.enabled` property on the organization; if it's not set, it returns null, which is what the dataset is providing.

Reply:
@cnovak On a non-hybrid org, the property `features.hybrid.enabled` is not present, so `$organization->getPropertyValue('features.hybrid.enabled')` would return null, which is what the data provider is returning:
```
// Format: ['features.isCpsEnabled', 'features.hybrid.enabled', $paginationAvailable, $message]
public function paginationAvailableValueProvider(): array
    {
        return [
            [null, 'true', true, 'Hybrid orgs should have pagination.'],
            ['true', null, true, 'CPS enabled orgs should have pagination.'],
            ['false', null, false, 'Non-Hybrid org without CPS should not have pagination.'],
        ];
    }
```